src/pile.js

```diff
@@ -500,13 +501,18 @@ const createPile = (
   };
 
   const positionItems = (
-    itemAlignment,
+    pileItemOffset,
     itemRotation,
     animator,
     previewSpacing
   ) => {
     isPositioning = true;
     let angle = 0;
+
+    const offset = isFunction(pileItemOffset)
+      ? pileItemOffset(store.getState().piles[id])
```

Comment:
This should to be called for every item on a pile and the signature should be `pileItemOffset(itemState, itemIndex, pileState)`